model/QueueBroker/RdsQueueBroker.php

```diff
@@ -34,6 +39,8 @@
  */
 class RdsQueueBroker extends AbstractQueueBroker
 {
+    use UuidPrimaryKeyTrait;
```

Comment:
not able to find this class

Reply:
i think @julien-sebire modified that function 
`return strrev(uniqid('', true));``
because it will collide, but ok, I think I am not on that branch while i checked the code of your development

Reply:
Indeed, there are should UUIDv4 generator, that exists somewhere in other dependent branch